packages/x-components/cypress.config.ts

```diff
@@ -0,0 +1,43 @@
+import { addCucumberPreprocessorPlugin } from '@badeball/cypress-cucumber-preprocessor';
+import { createEsbuildPlugin } from '@badeball/cypress-cucumber-preprocessor/esbuild';
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:8080',
+    defaultCommandTimeout: 7000,
+    viewportHeight: 1080,
+    viewportWidth: 1920,
+    screenshotOnRunFailure: false,
+    video: false,
+    supportFile: 'tests/support/index.ts',
+    fixturesFolder: 'tests/e2e/fixtures',
+    screenshotsFolder: 'tests/e2e/screenshots',
+    retries: {
+      openMode: 0,
+      runMode: 1
+    },
+    async setupNodeEvents(on, config) {
+      await addCucumberPreprocessorPlugin(on, config);
+      on(
+        'file:preprocessor',
+        require('@bahmutov/cypress-esbuild-preprocessor')({
+          plugins: [createEsbuildPlugin(config)]
+        })
+      );
+      return config;
+    },
+    specPattern: 'tests/e2e/**/*.feature'
+  },
+  component: {
+    specPattern: 'tests/unit/**/*.spec.ts',
+    supportFile: 'tests/support/index.ts',
+    indexHtmlFile: 'tests/support/component-index.html',
```

Comment:
**nitpick**: this is the default config value, could we remove it here? ([docs reference](https://docs.cypress.io/guides/component-testing/component-framework-configuration#Custom-Index-File)).

```suggestion
```

Reply:
This is not the default one, the default is: `cypress/support/component-index.html`
https://docs.cypress.io/guides/component-testing/component-framework-configuration#Custom-Dev-Server

Reply:
you're right, my bad!